src/components/inputs/Search/SearchBar/SearchBar.tsx

```diff
@@ -123,12 +117,15 @@ function SearchBar(props: SearchBarProps) {
       ) : (
         <></>
       )}
+
       <div
         className="searchbar"
         style={
-          isSearchValid
-            ? !searchSubmitted || showDefaultText
-              ? { borderColor: '' }
+          !searchBarText
+            ? { borderColor: '', marginBottom: 30 }
+            : isSearchValid
+            ? !searchSubmitted
+              ? { borderColor: 'white', marginBottom: 30 }
```

Comment:
really hard to read that. Is not easy to have a `const` that will evaluate the right styles and use it here?

Reply:
im confused, do you mean do the same thing but move it to a const?

Reply:
yeah - i think he's suggesting for such a complicated style rule, move it to a function or constant and avoid the nested ternary operators (i agree)

Reply:
fixed in https://github.com/ar-io/pdns-react/pull/227/commits/ef1c3d28bdea0b95c3fceabe861cf6142082e488

Reply:
Great, thanks!